examples/5. Example Floating Project-SemiTaut.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
I'm leaving a few comments here on this notebook since the diff view is useless:
- Be sure to restart the kernel and run all cells for notebooks to demonstrate they work as intended
- It'd be great if there was any narrative driving this example as it's not self-evident (though I recognize this is repo-wide issue)
- In cell under "Top Level Outputs", the f-strings should be ",.0f" so that large numbers show more cleanly, and a "$" should be added in front of the numbers since they're monetary values
- I'm not sure what "Installation Actions" is showing at all.